### PR TITLE
feat(events): Fall fest 2023

### PR DESCRIPTION
**Pull request by y4izus**
## Changes

Fix https://github.com/Qiskit/qiskit.org/issues/3517 and https://github.com/Qiskit/qiskit.org/issues/3518

## Implementation details

On the user story there are other 2 issues, but they will be done in the future

### CODE REVIEW

File: constants/fallFest2023Content.ts [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+
+const header = {
+  titleLine1: "Qiskit Fall Fest 2023",
+  description:
+    "The Qiskit Fall Fest is a collection of quantum computing events on college campuses around the world. Every Fall Fest event is organized and run by the students on each campus, with support from IBM Quantum. Student leaders work alongside IBM to design and plan their event, which are run in October and November. Each event is standalone - only made for the students on that particular campus. The events vary from technical hackathons to beginner-friendly challenges, all with the goal of helping to grow the local community’s skill set.",
+  cta: {
+    label: "2022 Fall Fest events",
+    url: "/events/fall-fest-2022",
+    segment: {
+      cta: "learn",
```
Comment (y4izus) [inline constants/fallFest2023Content.ts:11]: I don't think `learn` is a good cta, we need to replace it with a better one

File: constants/fallFest2023Content.ts [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+    title: "Qiskit Fall Fest 2023",
+    description:
+      "The Qiskit Fall Fest is a collection of quantum computing events each held by a university!",
+    location: "",
+    time: "",
+    to: "https://medium.com/qiskit/introducing-the-qiskit-fall-fest-feb8456b557",
+    ctaLabel: "Learn about the Fall Fest",
+    segment: {
+      cta: "qiskit-fall-fest-blog",
+      location: "header",
```
Comment (y4izus) [inline constants/fallFest2023Content.ts:27]: I would specify that is the card of the header. Something like

```suggestion
      location: "card-header",
```

**Review by y4izus — COMMENTED** [review]
One more thing, the card image on the header cut the text on mobile:

Captura de pantalla 2023-09-04 a las 14 43 43

File: constants/fallFest2023Content.ts (hunk 1 above)
Comment (techtolentino) [reply constants/fallFest2023Content.ts]: addressed in https://github.com/Qiskit/qiskit.org/pull/3530/commits/49621699296972dc357619bb777e36459495df76

**Comment by techtolentino** [comment]
> One more thing, the card image on the header cut the text on mobile:
> 
> Captura de pantalla 2023-09-04 a las 14 43 43


### Screenshot
Addressed the cut off text here

https://github.com/Qiskit/qiskit.org/assets/6276074/e66eb46c-b8b2-4d49-ad41-0269617238c5

**Review by techtolentino — APPROVED** [review]
Approved by the team

**Pull request merged**